org/pr/android-library.ts

```diff
@@ -0,0 +1,30 @@
+import {fail, danger} from "danger";
+
+// We want each PR towards develop or trunk of our Android libraries to bump
+// the library version. This check makes sure of it.
+export default async () => {
+  const baseBranch = danger.github.pr.base.ref
+  if (baseBranch != "develop" && baseBranch != "trunk") { return }
+
+  // Android projects have more than one build.gradle file, to be able to run
+  // this check on any library, lets's just look into _all_ the build.gradle
+  // files.
+  const modified = danger.git.modified_files
+    .filter((path: string) => path.endsWith('build.gradle'))
+
+  for (let file of modified) {
```

Comment:
I would have preferred not to use a for loop, I'm more into collection operations, but I didn't know how to write one because of the `await` inside.

Anyways, this does the job and it's quite tidy, too.